app/components/SignUp/UserInfo.js

```diff
@@ -121,11 +123,25 @@ export class UserInfo extends React.Component {
               required
             />
           </div>
-          <Submit
-            label={t('common:actions.basic.submit')}
-            disabled={pending}
-            processing={pending}
-          />
+          <div className="NavButtons">
+            <div className="hxRow">
+              <div className="hxCol hxSpan-6 align-left">
+                <Button
+                  classNames="btn-wide"
+                  onClick={() => history.push('/address')}
+                  label={t('common:actions.basic.back')}
+                />
+              </div>
+              <div className="hxCol hxSpan-6 align-right">
+                <Submit
+                  classNames="hxBtn hxPrimary"
+                  label={t('common:actions.basic.submit')}
+                  disabled={pending || !valid}
+                  processing={pending}
+                />
+              </div>
+            </div>
+          </div>
```

Comment:
grids vs regular old CSS...CSS might just be simpler.
```suggestion
          <div className="NavButtons">
                <Button
                  classNames="btn-wide u-pull-left"
                  onClick={() => history.push('/address')}
                  label={t('common:actions.basic.back')}
                />
                <Submit
                  classNames="hxBtn hxPrimary u-pull-right"
                  label={t('common:actions.basic.submit')}
                  disabled={pending || !valid}
                  processing={pending}
                />
          </div>
```